homeassistant/components/vacuum/xiaomi_miio.py

```diff
@@ -344,8 +363,14 @@ def async_update(self):
         from mirobo import DeviceException
         try:
             state = yield from self.hass.async_add_job(self._vacuum.status)
+            consumable = yield from self.hass.async_add_job(
```

Comment:
You can just set the instance attribute `self.consumable_state` directly.

Reply:
Thanks. I will change that tonight.